projects/vdk-plugins/airflow-provider-vdk/tests/hooks/test_vdkhook.py

```diff
@@ -1,9 +1,48 @@
 # Copyright 2021 VMware, Inc.
```

Comment:
I don't think the work `hook` should be included in the name of file.

Reply:
I've been following the airflow-provider-sample and it includes it https://github.com/astronomer/airflow-provider-sample/blob/main/tests/hooks/test_sample_hook.py

Reply:
Indeed, the sample tests use the word hook. However, this does not seem the case for the existing community providers https://github.com/apache/airflow/tree/main/tests/providers. I guess we can leave it as is.